protocol/messenger_local_backup.go

```diff
@@ -0,0 +1,196 @@
+package protocol
+
+import (
+	"context"
+	crand "crypto/rand"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/golang/protobuf/proto"
+	"go.uber.org/zap"
+
+	gocommon "github.com/status-im/status-go/common"
+	"github.com/status-im/status-go/protocol/common"
+	"github.com/status-im/status-go/protocol/protobuf"
+	v1protocol "github.com/status-im/status-go/protocol/v1"
+)
+
+// localBackupInterval is the duration we should allow between backups
+var localBackupInterval = 1800 * time.Second // 30 minutes
+
+func (m *Messenger) startLocalBackupLoop() {
+	ticker := time.NewTicker(localBackupInterval)
```

Comment:
Better this way then stopping it in `<-m.quit`
```suggestion
	ticker := time.NewTicker(localBackupInterval)
	defer ticker.Stop()
```